### PR TITLE
Update Webdriver for nanoid and fix other security issues

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
Webdriver needed to be updated to v7 to fix a security issue a couple of layers down the dependancy tree.

- nanoid needed updating to v3.1.31 or later for the security issue
- nanoid is a dependancy of mocha
- mocha is a dependancy of mocha-framework which is part of webdriver

This PR will update Webdriver to the latest version (7.16.16) and its dependancies will now use nanoid v3.2.0.

This PR also fixes a number of other vulnerabilities highlighted by running `yarn audit`. Because of this these packages have also been updated:
- uuid
- node-jose
- eslint

For compatibility babel also needed to be updated to v7 and some changes needed to be made to the config because of this.

This also updates the node version in the github action to be the same as the one in the .nvmrc file

I have also upgraded all packages highlighted by running `yarn outdated` bar `eslint-config-standard` and `eslint-plugin-standard` as there are some compatibility issues around these with v8 of eslint, waiting on v17 of eslint-config-standard. Once that is released and we update to that we can remove `eslint-plugin-standard` as its no longer needed.

### How to review 
- Run `yarn audit` on master
- Run `yarn audit` on this branch and see all vulnerabilities are fixed
- Run `yarn outdated` on master
- Run `yarn outdated` on this branch and see nearly all packages are updated
- Functional tests pass

### Checklist

* [ ] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by rmccar** [comment]
[Update webdriver for security issue](https://trello.com/c/dVH9czRh/5196-update-webdriver-for-security-issue)

**Review by parimooa — APPROVED** [review]
All functional tests passing and no vulnerabilities found

File: tests/functional/jwt_helper.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import KJUR from "jsrsasign";
-import uuid from "uuid/v1";
+import { v1 as uuidv1 } from "uuid";
```
Comment (MebinAbraham) [inline tests/functional/jwt_helper.js:2]: We should use v4 as that is what runner actually uses.

**Comment by rmccar** [comment]
> Reminds me that, package.json only pins to the latest minor versions. At some point soon it would be good to updated to the latest major for all.

The packages I have updated in this PR are all updated the latest versions

**Comment by rmccar** [comment]
> Reminds me that, package.json only pins to the latest minor versions. At some point soon it would be good to updated to the latest major for all.

I have now updated all the packages I can at the moment apart from the 2 I have mentioned in the PR description

**Review by MebinAbraham — APPROVED** [review]
👍

**Pull request merged**